server/methods/core/orders.js

```diff
@@ -213,6 +213,63 @@ export const methods = {
     });
   },
 
+
+  /**
+   * orders/cancelOrder
+   *
+   * @summary Start the cancel order process
+   * @param {Object} order - order object
+   * @param {Boolean} returnToStock - condition to return product to stock
+   * @return {Object} ret
+   */
+  "orders/cancelOrder": function (order, returnToStock) {
+    check(order, Object);
+    check(returnToStock, Boolean);
+
+    if (!Reaction.hasPermission("orders")) {
+      throw new Meteor.Error(403, "Access Denied");
+    }
+
+    if (!returnToStock) ordersInventoryAdjust(order._id);
```

Comment:
While this is syntactically correct, I prefer would us to be consistent and use the full `if/then` syntax